xtask/tests/tidy.rs

```diff
@@ -50,6 +50,20 @@ fn rust_files_are_tidy() {
     tidy_docs.finish();
 }
 
+#[test]
+fn check_merge_commits() {
+    let merge_head = run!("git merge-base HEAD origin/master"; echo = false).unwrap();
+    let cmd_output = run!("git rev-list --merges --invert-grep --author 'bors[bot]' {}...", merge_head; echo = false);
```

Comment:
Why do we need merge base here? Wouldn't just `git rev-list -n 5 HEAD` work?

Reply:
There may be a merge hidden before 5 or more commits.

Reply:
<img width="573" alt="Screenshot 2020-08-28 at 9 15 36 AM" src="https://user-images.githubusercontent.com/6277978/91583927-1ab3e580-e90f-11ea-80f4-280583842cd5.png">
I need to know the `HEAD` for `origin/master` so that it can traverse from there by `rev-list`. No, it doesn't work as you can see from the screenshot. I don't know if there's a better way to do this!

Reply:
>I need to know the `HEAD` for `origin/master` so that it can traverse from there by `rev-list`. 

I don't quite understand why you need to know anything about `origin/master` though. I would think that we can traverse from the current commit backwards, and that we don't really need to know the state of master for that?

Reply:
let's try that!

Reply:
The `-n 5` will only limit the amount of results that `git rev-list` returns, not the amount of history it searches. This means that if the history is not just 5 commits deep (for example when running the test locally), you'd always get 5 results. To limit the history that is searched, you'd have to do something like `git rev-list <flags> HEAD~5..`, limiting the search to commits since `HEAD~5` e.g. 5 commits ago

Reply:
Nice catch.  I am learning a lot about Git in this PR. Thanks!